lang/rust/rust.py

```diff
@@ -0,0 +1,185 @@
+from talon import Module, Context, actions, ui, imgui, settings
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r'''
+mode: user.rust
+mode: user.auto_lang
+and code.language: rust
+'''
+
+ctx.tags = ['user.code_operators', 'user.code_generic']
```

Comment:
Why not set these tags in the `.talon` file along with the rest?

... actually, do these tags even still exist?

Reply:
I don’t even know whether they exist… It’s probably copy paste from somewhere, frankly.